src/System.Private.CoreLib/shared/Internal/Win32/RegistryKey.cs

```diff
@@ -362,7 +363,7 @@ public object GetValue(string name, object defaultValue)
                                 throw new IOException(SR.Arg_RegGetOverflowBug, e);
                             }
                         }
-                        char[] blob = new char[datasize / 2];
+                        char[]? blob = new char[datasize / 2];
```

Comment:
Why is this nullable?

Reply:
same reason Array.Resize takes a nullable array as argument

Reply:
Same response :)